src/pages/PageNotFound.jsx

```diff
@@ -0,0 +1,26 @@
+import styles from './PageNotFound.module.scss';
+
+function PageNotFound() {
+  let option = Math.random() > 0.5;
+  let image = option ? '/images/sherlock.png' : '/images/shakespeare.png';
+  let text = option
+    ? "Even Sherlock Holmes couldn't find this page.~"
+    : 'Thou hast stumbled upon an uncharted kingdom of 404!~';
+
+  return (
+    <>
+      <div className={`${styles.pageNotFound}`}>
+        <div className={`${styles.textColumn}`}>
+          <div className={`${styles.errorCode}`}> 404 </div>
+          <div className={`${styles.sorryText}`}> SORRY! </div>
+          <div>{text}</div>
+        </div>
+        <div className={`${styles.imageColumn}`}>
```

Comment:
```suggestion
      <div className={styles.pageNotFound}>
        <div className={styles.textColumn}>
          <div className={styles.errorCode}> 404 </div>
          <div className={styles.sorryText}> SORRY! </div>
          <div>{text}</div>
        </div>
        <div className={styles.imageColumn}>
```

I think string interpolations are not needed in those cases. But I'm not sure, please test it if it works or not. Thanks